src/settings.js

```diff
@@ -856,6 +856,13 @@ var FETCH = 0; // If nonzero, enables emscripten_fetch API.
 
 var ASMFS = 0; // If set to 1, uses the multithreaded filesystem that is implemented within the asm.js module, using emscripten_fetch. Implies -s FETCH=1.
 
+var SINGLE_FILE = 0; // If set to 1, embeds all subresources in the emitted JS file
```

Comment:
please add examples of those files (data, asm.js, wasm etc.), i think it would help clarify things